llvm/lib/Transforms/Instrumentation/AddressSanitizer.cpp

```diff
@@ -476,6 +489,9 @@ struct ShadowMapping {
 
 } // end anonymous namespace
 
+constexpr unsigned OffloadGlobalAS = 1;
```

Comment:
To differentiate from e.g. AMDGPU offload
```suggestion
constexpr unsigned SpirOffloadGlobalAS = 1;
```

Reply:
Good suggestion. Done.